wandb/sdk/internal/sender.py

```diff
@@ -606,12 +607,21 @@ def send_artifact(self, data):
 
     def send_alert(self, data):
         alert = data.alert
-        self._api.notify_scriptable_run_alert(
-            title=alert.title,
-            text=alert.text,
-            level=alert.level,
-            wait_duration=alert.wait_duration,
-        )
+        _, server_info = self._api.viewer_server_info()
+        if version.parse(
+            server_info["cliVersionInfo"]["max_cli_version"]
+        ) < version.parse("0.10.9"):
+            wandb.termwarn(
```

Comment:
termwarn should not be called from the internal process.

Reply:
@raubitsj Is it sufficient to just log this as a warning in the logs? I'm not sure how else to print a message to the user due to the potentially asynchronous flow when offline runs are later synced.

Reply:
Piping stuff through is a little tricky today.  Simplest solution is to do this logic in the `wandb_run` object when the user method is called.

Reply:
That breaks offline support though, because then we need to ping the server at least once to establish the version. I guess we can disable the check in offline mode.

Reply:
That doesnt handle the offline sync case.

I like the log approach for now.

The proper way is likely to return an error in the proto response, but if this isnt synchronous, then we dont have a good solution.

Reply:
I switched it to logs. The response doesn't really work because then that assumes alerts are synchronous.